rakefile-dotnet.rb

```diff
@@ -258,6 +268,12 @@
   cmd.parameters "pack #{nuspec} -Version #{version}#{version_suffix} -OutputDirectory #{output}"
 end
 
+desc "create the nuget package with the .NET Standard"
+exec :pack => [:build, output] do |cmd|
+  cmd.command = nuget_command
+  cmd.parameters "pack #{dotnet_nuspec} -Version #{version}-dotnet#{version_suffix} -OutputDirectory #{output}"
+end
+

```

Comment:
> I'm more than happy to switch to `netstd` as the moniker everywhere, but we seem to be delaying that and going with `dotnet` everywhere for now, which is why I suggested `_DOTNET`. I think we should do one or the other, but not mix.

I'm happy with netstd. I didn't realize we were delaying as such.
I haven't converted my PR yet, but am happy to do so the second all the project names and paths and whatnot switch, but it's a separate task from "make a rakefile".
I could take a crack at the conversion to netstd, if nobody else wants to, but otherwise, I'm happy to continue on with this task.

> The problem is, we'll need to be able to control the version suffix for the net40 package independently of the net40+netstd package. E.g. when we are pushing out an RC of the net40 package, I don't think we'll want to use RC in the version suffix of the net40+netstd package.

I don't really see any harm. We could either not publish the RC package or publish and then never publish the production release with the same version number. 

> And likewise, when we push out net40 RTM, we'll want the net40+netstd to still have a pre-release suffix.

Sure! "-netstd". It satisfies [point 9 of SemVer](Satisfies http://semver.org/#spec-item-9)'s requirements for being a 'pre-release' version. And is more intention-revealing than "-beta".